styleguide/examples/sample_pages/form-sample.html

```diff
@@ -96,6 +96,7 @@
               id="search-query"
               aria-label="Search through site content"
               class="cads-search__input cads-input"
+              autocomplete="off"
```

Comment:
Adding `autocomplete="off"` to the search input was recommended by erblint which seems legit so have added this into the template. Nice work linter.